CRM/MembersOnlyEvent/BAO/MembersOnlyEvent.php

```diff
@@ -75,7 +96,7 @@ public static function getMembersOnlyEvents($eventIDs) {
 
     $result = civicrm_api3('MembersOnlyEvent', 'get', [
       'sequential' => 1,
-      'is_groups_only' => 1,
+      'event_access_type' => 2,
```

Comment:
Should we use here the constants that you created? `EVENT_ACCESS_TYPE_GROUPS_ONLY`

Reply:
Updated to pass it as a parameter instead of hard coding it,  see the commit '[WHF-308: Refactor the method getMembersOnlyEvents to use event access type'](https://github.com/compucorp/uk.co.compucorp.membersonlyevent/pull/30/commits/68ee388fffdcf7c7baf5becbd2a5562d178d8c4d)